build-logic/settings.gradle.kts

```
@@ -0,0 +1,5 @@
+rootProject.name = "build-logic"
+
+plugins {
+    id("dev.panuszewski.typesafe-conventions") version "0.9.0"
```

nit: [0.10.0](https://github.com/radoslaw-panuszewski/typesafe-conventions-gradle-plugin/releases/tag/v0.10.0) is available.


```suggestion
    id("dev.panuszewski.typesafe-conventions") version "0.10.0"
```